api/pkg/s3/bucketget.go

```diff
@@ -123,10 +123,19 @@ func parseListObjectsQuery(query url.Values) (request s3.ListObjectsRequest, err
 	return
 }
 
+func CreateListOfListObjectsResponse(bucketName string, request *s3.ListObjectsRequest,
+	listRsp *s3.ListObjectsResponse) (response []datatype.ListObjectsResponse) {
+	for _, o := range listRsp.ListOfListOfObjects {
```

Comment:
The listRsp shld have a validatikn and logging

Reply:
Will add in testing phase, after RC4 during code coverage and functional testing

Reply:
RC4 is the final one. This is the final release cut :)